### DISCUSSION TITLE
Question on deploying to branch name

### DISCUSSION THREAD

**Discussion by costateixeira** [JamesIves/github-pages-deploy-action#764] [Q&A]
Hi

I'd like to deploy some selected branches to a folder named as the branch: 
When I commit to rc1, it'd deploy to gh-pages/rc1
When I commit to my-branch, it'd deploy to gh-pages/my-branch

I tried to set
```
 target-folder: draft:  ${GITHUB_REF##*/}
```
but that doesn't work (it creates a folder called literally `gh-pages/${GITHUB_REF##*/}`

Is there a way to pass the branch name as a folder to deploy to?

**Comment by JamesIves** [comment]
I think you're looking for `${{ github.ref }}`.

Docs: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions
